### DISCUSSION TITLE
scRNA-seq integration is only partial

### DISCUSSION THREAD

**Discussion by LacquerHed** [GreenleafLab/ArchR#1650] [Questions / Documentation]
When I perform the integration (using defaults) only some of the scRNA-seq clusters get mapped onto my scATAC-seq clusters, I am wondering which particular variables/commands I can play around with to increase the proportion of integrated clusters? Thanks.

**Comment by rcorces** [comment]
Beyond the parameter definitions and the information provided in the manual, I do not have additional advice. If you think something in particular is missing from the provided documentation, please let me know.
https://www.archrproject.com/reference/addGeneIntegrationMatrix.html
https://www.archrproject.com/bookdown/cross-platform-linkage-of-scatac-seq-cells-with-scrna-seq-cells.html

**Reply by LacquerHed** [reply]
A more specific question would be is if you're sampling 10K cells for integration from the RNA dataset which is considerably larger, would that perhaps lead to reduced integration of rare cell types?

**Reply by rcorces** [reply]
seems possible to me
